src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -154,6 +154,34 @@ def stopped(self) -> None:
         logging.info(PROXY_LOG_PREFIX + "removing proxy: %s", self.region)
         self.delete()
 
+    def is_outdated(self) -> bool:
+        if self.version != __version__:
+            logging.info(
+                PROXY_LOG_PREFIX + "mismatch version: proxy:%s service:%s state:%s",
+                self.version,
+                __version__,
+                self.state,
+            )
+            return False
+        if self.timestamp is not None:
+            proxy_timestamp = self.timestamp
```

Comment:
I think I'm realizing during testing that this self.timestamp value represents the last time the proxy sent out a heartbeat. I think this means I need to create a new field that represents 'created_timestamp'.